hpat/hiframes/hiframes_typed.py

```diff
@@ -879,6 +879,9 @@ def _run_call_series(self, assign, lhs, rhs, series_var, func_name):
                 data = self._get_series_data(series_var, nodes)
             return self._replace_func(func, [data], pre_nodes=nodes)
 
+        if func_name == 'median':
+            return [assign]
```

Comment:
You will achieve the line never, because `'median'` is processed above. Moreover if `func_name` doesn't match any condition in this function `return [assign]` is by default.

Reply:
@densmirn Yes, it was needed only when 'median' was removed from the tuple above. 
It'll be removed in the next commit.